lib/std/strutils.nim

```diff
@@ -328,6 +328,26 @@ func cmpIgnoreStyle*(a, b: string): int =
     inc i
     inc j
 
+type
+  Natural = int
```

Comment:
Move this alias to system.nim

Reply:
It is done in https://github.com/nim-lang/nimony/pull/1415